### DISCUSSION TITLE
Add GMA Regional Local Channel (Not 24/7)

### DISCUSSION THREAD

**Discussion by Primo-coder** [iptv-org/community#1139] [Q&A]
I want to add and watch
GMA Davao
GMA Cebu
GMA Iloilo
GMA Dagupan

Because I want to watch GMA Regional Local News

(Sadly GMA Regional Axed Balitang southern tagalog and Balitang Bicolandia)

Please add 4 GMA Regional Local Channel

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
